jax/api.py

```diff
@@ -90,6 +90,9 @@ def jit(fun, static_argnums=()):
       different values for these constants will trigger recompilation. If the
       jitted function is called with fewer positional arguments than indicated
       by `static_argnums` then an error is raised. Defaults to ().
+    device_assignment: Optional, an int specifying the device ordinal for which
```

Comment:
Just a caution: I think it highly likely that we will stop using ints to name devices here soon (e.g., it doesn't necessarily work that well in a multihost setup). I'm not sure whether you wish to add a caveat to the documentation that this API is not yet stable.

Reply:
Great point!